### PR TITLE
docs - isnull and isempty

### PR DESCRIPTION
**Pull request by nllho**
Putting up `isnull` and `isempty` together because they're so similar (and contain references to one another). 

I put in a [suggestion](https://www.notion.so/metabase/Link-to-targeted-custom-expression-docs-when-available-c16afcfa28134e7f83a4078bcd2c1689) to link to these custom expressions pages as they go out.

- Is it easier to organize everything in an expressions directory _before_ we share those links with product (to avoid breaking them in the near future)?
- If so, do we want the path to be `/questions/expressions`, or something else?

### CODE REVIEW

File: docs/users-guide/expressions/isempty.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+Combine `isempty` with the [`case` expression](./case) to replace empty strings with something more descriptive.
+
+If the second row's blank cell is actually an empty string, then `isempty` will return `true`. The `case` statement evaluates `true` to return the first output "No feedback".
+
+If the first row's blank cell is actually a `null`, (or even an emoji that blends into your table background), then `isempty` will return `false`, and `case` will return whatever's in the Feedback column as the default output.
+
+## Accepted data types
+
+| [Data type][data-types] | Works with `isempty` |
```
Comment (jeff-bruemmer) [inline docs/users-guide/expressions/isempty.md:47]: Do we need a table here?

File: docs/users-guide/expressions/isempty.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+| Feedback           | `case(isempty([Feedback]), "No feedback.", [Feedback])` |
+| ------------------ | ------------------------------------------------------- |
+|                    |                                                         |
+|                    | No feedback.                                            |
+| I like your style. | I like your style.                                      |
+
+Combine `isempty` with the [`case` expression](./case) to replace empty strings with something more descriptive.
+
+If the second row's blank cell is actually an empty string, then `isempty` will return `true`. The `case` statement evaluates `true` to return the first output "No feedback".
```
Comment (jeff-bruemmer) [inline docs/users-guide/expressions/isempty.md:41]: Should we drop the "ifs" here? We should be describing  why the table above is the way it is.

File: docs/users-guide/expressions/isnull.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+Combine `isnull` with the [`case` expression](./case) to replace "unknown" information with something more descriptive.
+
+If the first row's blank cell is actually a `null`, then `isnull` will return `true`. The `case` statement evaluates `true` to return the first output "Unknown feedback".
+
+If the second row's blank cell is actually an empty string (or even an emoji that blends into your table's background), then `isnull` will return `false`, and `case` will return whatever's in the Feedback column as the default output.
+
+## Accepted data types
+
+| [Data type][data-types] | Works with `isnull` |
```
Comment (jeff-bruemmer) [inline docs/users-guide/expressions/isnull.md:45]: Maybe a list here?

**Review by jeff-bruemmer — APPROVED** [review]
"I like your style", @nllho. Just a few comments.

File: docs/users-guide/expressions/isempty.md (hunk 1 above)
Comment (nllho) [reply docs/users-guide/expressions/isempty.md:47]: Gonna keep it, based on the assumption that people tend to scan these types of docs, so consistency is key.

If we make it easy to scan for "all green", it should be easier to catch people's attention when there are "reds" (hopefully it'll make them pause and read "Limitations" below).

Comment (nllho) [reply docs/users-guide/expressions/isempty.md:47]: For posterity -- the hypothesis that came out of our discussion :)

> I’m also thinking about the case where people can’t tell the difference between the actual data type vs. the data types mapped to a string.
For example, if someone knows that a function “works with strings” and “doesn’t work with numbers”, a list of all of the data types can lead to the right question — “is my number somehow being typed as a string?”
Without that information, the person doesn’t have as much of a lead as to what’s going wrong.

**Pull request merged**